src/core/stream/index.ts

```diff
@@ -398,14 +402,39 @@ export default function Stream({
       return Observable.of(evt);
     });
 
-    /**
-     * Add management of events linked to live Playback.
-     * @type {Observable}
+    /* In the case where minimumUpdatePeriod is specified, and above 0,
+     * manifest may be updated when update period has elapsed from the download
+     * time of previous manifest.
+     * @param {number} minimumUpdatePeriod
+     * @param {number} updateGap
      */
-    const buffers$ = (manifest.isLive ?
-      handledBuffers$
-        .mergeMap(liveEventsHandler(videoElement, manifest, fetchManifest)) :
-      handledBuffers$);
+    function refreshExpiredManifest(
+      updateGap: number,
+      minimumUpdatePeriod?: number
+    ): Observable<IManifestUpdateEvent> {
```

Comment:
Wouldn't it be cleaner for it to take the manifest in argument and perform the:

```js
Date.now() / 1000 - manifest.loadedAt
```

calculation itself?